templates/config/newrelic.yml

```diff
@@ -0,0 +1,61 @@
+#
+# This file configures the New Relic Agent.  New Relic monitors Ruby, Java,
+# .NET, PHP, Python, Node, and Go applications with deep visibility and low
+# overhead.  For more information, visit www.newrelic.com.
+#
+# Generated January 12, 2022
+#
+# This configuration file is custom generated for NewRelic Administration
+#
+
+common: &default_settings
+  # Required license key associated with your New Relic account.
+  license_key: <%= Rails.application.credentials.new_relic_key %>
+  # FEDRAMP-specific New Relic host
+  # https://docs.newrelic.com/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/
+  host: 'gov-collector.newrelic.com'
+
+  # Your application name. Renaming here affects where data displays in New
+  # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+  app_name: <APPNAME>
+
+  monitor_mode: true
+
+  distributed_tracing:
+    enabled: true
+
+  browser_monitoring:
+    # include js code via partial to comply with CSP settings
+    auto_instrument: false
+
+  # To disable the agent regardless of other settings, uncomment the following:
+  # agent_enabled: false
```

Comment:
[suggestion] why don't we uncomment this line (and update the accompanying explanation) to better deal with the chicken and egg problem of including this file before we have a license key and app name?

Reply:
I like that approach! will add an update